environment

```diff
@@ -159,6 +153,7 @@ _set AZUL_SUBDOMAIN_TEMPLATE "{lambda_name}"
 _set AZUL_RESOURCE_PREFIX "azul-"
 _set AZUL_ES_INDEX "azul-$AZUL_DEPLOYMENT_STAGE"
 _set AZUL_ES_DOMAIN "azul-index-$AZUL_DEPLOYMENT_STAGE"
+_set AZUL_ES_INDEX_NAME "azul_{index_name}_$AZUL_DEPLOYMENT_STAGE"
```

Comment:
s/index_name/entity_type

Reply:
But the values stay as is (plural, for backwards compatibility). E.g. `files`, `specimen`.